{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "python3",
                "purl": "pkg:rpm/vmware/python3?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "3.6.9-14.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2022-0516",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2022-09-15T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-2.0-516"
        }
    ],
    "related": [
        "CVE-2021-28861",
        "CVE-2021-4189"
    ]
}